templates/activity.html

```diff
@@ -1,41 +1,29 @@
-<!doctype html>
-<html>
+{% extends 'includes.html' %}
+{% block content %}
+    <div class="section no-pad-bot" id="index-banner">
+        <div class="container">
```

Comment:
This `div` is all a repeat of what is on `index.html` ?

Reply:
no. it generates code something like http://paste.ubuntu.com/26200076/

Reply:
But why?
This page should have the standard header and footer and no extra stuff.  It is just a page, like any other.  The activity page is not the main page.